README.md

```diff
@@ -8,15 +8,110 @@
 
 ## Overview
 
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+[Jira](https://www.atlassian.com/software/jira)  is a widely-used issue and project tracking software by Atlassian. It provides a REST API that allows applications to connect with Jira to create, update, and delete various resources, enabling efficient project and task management.
+
+The `ballerinax/jira` package offers APIs to interact with the [Jira REST API](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro), providing access to various endpoints for managing issues, filters, projects, and other resources.
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Jira connector, you must have a Jira Cloud account with API access. If you do not have one, you can sign up for a Jira account on [Atlassian’s website](https://developer.atlassian.com/cloud/jira/platform/).
+
+### Step 1: Create an API Token
+1. Login to [Atlassian Account](https://developer.atlassian.com/cloud/jira/platform/)
+
+2. Click on the profile icon to get into developer console
+
+![{D8EEDDDF-5D02-4907-A10A-911BB513050F}](https://github.com/user-attachments/assets/3ec0f5be-6c5f-4038-a37a-ec8a2f6e2ccf)
+
+3. In the new window, click on the profile again and go to 'Manage Account'
+
+4. Click and go to security tab and scroll down to the bottom to generate API token by clicking 'Create and Manage API token'
+
+![{8E9B6CE3-B218-46F4-8481-5211FE443418}](https://github.com/user-attachments/assets/081db352-e5d1-4657-8f85-49f791881975)
+
+5. Give a name to the API token and copy and save it in a secret place
+
+![{2385E238-124F-4D5C-9446-836EF0EF56A2}](https://github.com/user-attachments/assets/d76705c0-2bf9-4f04-9d9a-1ec963d950e9)
```

Comment:
Could you please add these images in the folder path `docs/setup/resources` and refer them here?
You can refer how it is done in Twitter connector -> https://github.com/ballerina-platform/module-ballerinax-twitter/tree/main/docs/setup/resources

Reply:
Did this change